npm/angular/.eslintrc

```diff
@@ -1,8 +1,7 @@
 {
   "plugins": [
     "cypress",
-    "@cypress/dev",
-    "@angular-eslint/eslint-plugin"
```

Comment:
I removed this because running lint was throwing an error saying it couldn't find this dep. It's not really providing any value currently as we are just serving the default scaffold angular app that gets generated via `ng new`. We can always add in these or other angular eslint rules in the future